### ISSUE TITLE
blurry ness around corners

### ISSUE THREAD

**Issue by Aneeqasif** [Luwx/LightlyShaders#10]
there is wiered blurryness around corners no matter how much the value of roundness i set it to [check this screenshot](https://i.imgur.com/JCQWn6n.png)

**Comment by amidmajd** [comment]
I have this problem as well, for breeze theme only the default value for roundness works correctly!

**Comment by Luwx** [comment]
As I said in #7, the decoration must have the same corner radius as the effect. Some decorations have the option to configure the corner radius, therefore making the shadow "round" too.

**Comment by ryu-ketsueki** [comment]
> As I said in #7, the decoration must have the same corner radius as the effect. Some decorations have the option to configure the corner radius, therefore making the shadow "round" too.

It's a different issue. The #7 was about window decoration shadows not matching the round corner of the shader. I did check ClassikStyles and they do have a roundness option to adjust. I did that and the shadows were adapted to the one on LightlyShaders. However, the blur on the corners happen with all window decorations, including Lightly's own window decoration so it is a fault on either the shader or KWin's native blur feature behind the window not being able to be cropped or resized.

![image](https://user-images.githubusercontent.com/39316730/125490996-7a8eeccc-d60b-4c6a-951d-8632c6885c95.png)
